query-tuning/collector-workflow.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Setting up the Collector Workflow'
+backlink_href: /docs/query-tuning
+backlink_title: 'Query Tuning'
+---
+
+import CopyCodeBlock from '../components/CopyCodeBlock'
+import DocsBetaFeature from '../components/DocsBetaFeature'
+import { explainAnalyzeHelper } from '../components/helperSQL'
+
+<DocsBetaFeature feature="Query Tuning" />
+
+The Collector Workflow in Query Tuning workbooks optionally allows users with the [Tune Queries permission]()
+to execute arbitrary EXPLAIN ANALYZE commands directly from the pganalyze user interface, avoiding tedious
```

Comment:
(nitpicking, fine as is too) feels like "avoiding tedious copy and pasting" contains some emotional touch, where I'd like the docs to explain the fact. How about changing this to something like:

"...from the pganalyze user interface. This eliminates the need for manual copying and pasting of command outputs."

Reply:
+1 for "I'd like the docs to explain the fact.", thanks for calling that out.